src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
I'd love it if we could get to something stupid simple, like:
```
player.playlist();
player.next();
player.previous();
```
But if we can't get there, `player.playlist.func()` is ok.

The `player.playlist().item(index)` route adds this nasty complexity:
```js
// Add a playlist
player.playlist([...]);
// Get a ref to the playlist
var pl1 = player.playlist();
// Add a different playlist
player.playlist([...]);
// Call item(index) on the first playlist reference
pl1.item(2); // what happens? It's not obvious.
```